include/mbedtls/mbedtls_config.h

```diff
@@ -3760,6 +3760,9 @@
  */
 //#define MBEDTLS_PSA_KEY_SLOT_COUNT 32
 
+/* RSA OPTIONS */
+//#define MBEDTLS_RSA_GEN_KEY_MIN_SIZE            1024 /**<  Minimum RSA key size allowed in bits (Minimum possible value is 128 bits) */
```

Comment:
It might be clearer to users if this is not commented out by default. The minimum value would be 1024 either way, but if it's not explicitly set here, the user can't easily see this. (Alternatively, a comment could say e.g. "the default is 1024 if not set", but maybe simpler just to un-comment this).

Reply:
Should we also comment, and check, that the minimum key size must be an even number?

I guess a user could set this to e.g. 129 and then generate keys of size 130 or more without issues so maybe OK as-is. @gilles-peskine-arm ?